### PR TITLE
Fix for new pages

### PR DESCRIPTION
**Pull request by shirayu**

### CODE REVIEW

File: lib/Finance/Quote/YahooJapan.pm [hunk 1]
```diff
@@ -175,30 +175,27 @@ sub _convert_quote {
...
-            # for a stock code, register a duplicate quote with market letter
-            if ($sym =~ /^[0-9A-Z]{2}[0-9][0-9A-Z][0-9]?$/) {
-                my $pat = qr/(?:quote\/|code=)($sym\.[A-Z])/;
-                $e->look_down('_tag', 'a', 'href', $pat)->attr('href') =~ $pat;
-                $quotes{lc $1} = $quote if (defined $1);
-            }
-
-            # XXX destructive when a stock quote from other market already exists
-            $quotes{$sym} = $quote;
+    my @nodes =  $tree->findnodes('//*[@id="sr"]/div/ul/li/article');
```
Comment (yyyjajp) [inline lib/Finance/Quote/YahooJapan.pm:178]: Is the xpath necessary? Just
`$container->find('article')`
should work well.

Comment (shirayu) [reply lib/Finance/Quote/YahooJapan.pm:178]: I thouhgt that removing the following three lines and using the xpath would be more readable.
I leave it to the maintainer to decide which is better.

```perl
    my $container = $tree->look_down('id', 'sr');
    if (defined $container) {
...
   }
```

**Comment by LiosK** [comment]
Please take a look at my fix branch: https://github.com/LiosK/Finance--Quote--YahooJapan/compare/master...fix

Added a comment line to address the readability concern. Thanks.

**Comment by shirayu** [comment]
Thanks! It seems good!

**Pull request closed without merging**